src/main/java/org/apache/mesos/hdfs/Scheduler.java

```diff
@@ -602,16 +603,22 @@ private boolean reconciliationComplete() {
     @Override
     public void run() {
       log.info("Current persistent state:");
-      log.info(String.format("JournalNodes: %s", persistentState.getJournalNodes()));
-      log.info(String.format("NameNodes: %s", persistentState.getNameNodes()));
+      log.info(String.format("JournalNodes: %s %s", persistentState.getJournalNodes(),
+          persistentState.getJournalNodeTaskNames()));
+      log.info(String.format("NameNodes: %s %s", persistentState.getNameNodes(),
+          persistentState.getNameNodeTaskNames()));
```

Comment:
Just curious what this log output looks like now. Should we be using something other than a space ' ' to delimit/describe the two variables printed on each line?